server/models.py

```diff
@@ -329,6 +329,13 @@ class SpiritScore(ExportModelOperationsMixin("spirit_score"), models.Model):  #
         UCPerson, on_delete=models.CASCADE, related_name="msp", blank=True, null=True
     )
 
+    mvp_v2 = models.ForeignKey(
+        Player, on_delete=models.CASCADE, related_name="mvp_v2", blank=True, null=True
```

Comment:
ideally it should not casacde as then we will delete the spirit score also when we delete a player.

we should set to null if we delete player here